src/tgz_archiver.h

```diff
@@ -2,6 +2,7 @@
 #define CATA_SRC_MAP_TGZ_ARCHIVER_H
 #include <array>
 #include <chrono>
+#include <filesystem>
```

Comment:
Use our filesystem header filesystem.h or <ghc/fs_std_fwd.h>. Although given it already resolves fs::file_time_type, one or both are already transitively provided.

Reply:
Weird.  I don't know how this ever worked on C++14, but I will fix.

Reply:
It was already included through `"filesystem.h"` -> `<ghc/fs_std_fwd.hpp>` -> `<filesystem>`

https://github.com/CleverRaven/Cataclysm-DDA/blob/8286f4d7d8f43e1d3b79125ceb2270facf9e0caf/src/tgz_archiver.h#L8